Enhance email template example (#8076)
umbraco/UmbracoDocs: 17/umbraco-forms/developer/email-templates.md

@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<Umbraco.Forms.Core.Models.FormsHtmlModel>
```

> You now have a model that contains your Form fields, which can be used in your email HTML markup, along with the UmbracoHelper methods such as `Umbraco.TypedContent` and `Umbraco.TypedMedia`, etc.

Reviewer: ```suggestion
You now have a model with your Form fields. This can be used in your email HTML markup, along with the UmbracoHelper methods like `Umbraco.TypedContent` and `Umbraco.TypedMedia`, and so on.
```
